client/src/components/EditProfile/EditProfile.tsx

```diff
@@ -1,24 +1,53 @@
-import { TextField, Paper } from '@material-ui/core';
-import { Grid } from '@material-ui/core';
-import { Typography } from '@material-ui/core';
-import Button from '@material-ui/core/Button';
+import { TextField, Paper, Grid, Typography, Button, CircularProgress } from '@material-ui/core';
 import useStyles from './useStyles';
 import Sidebar from '../Sidebar/SideBar';
+import { useSnackBar } from '../../context/useSnackbarContext';
+import { Formik, FormikHelpers } from 'formik';
+import * as Yup from 'yup';
+import editProfile from '../../helpers/APICalls/editProfile';
 
 interface Props {
+  _id?: string;
   firstName?: string;
   lastName?: string;
-  gender?: string;
-  birthDate?: Date;
   email?: string;
-  phone?: string;
-  place?: string;
+  phone?: number;
+  city?: string;
   description?: string;
 }
 
 const EditProfile = (props: Props): JSX.Element => {
   const classes = useStyles();
+  const { updateSnackBarMessage } = useSnackBar();
 
+  const handleSubmit = (
+    { firstName, lastName, email, phone, city, description }: Props,
+    { setSubmitting }: FormikHelpers<Props>,
+  ) => {
+    const id = props._id;
+    if (!id) {
+      setSubmitting(false);
+      updateSnackBarMessage('Could not find profile');
+      return;
+    }
+    const profile = { firstName, lastName, email, phone, city, description };
+    editProfile(id, profile).then((data) => {
+      if (data.error) {
+        console.error({ error: data.error.message });
```

Comment:
please remove as we are already updating the snack bar w/ the error msg

Reply:
Good point!